Formula/kumactl.rb

```diff
@@ -15,6 +14,12 @@ class Kumactl < Formula
 
   depends_on "go" => :build
 
+  # Fix build on ARM
```

Comment:
Is this fixed in the next version? If so:
```suggestion
  # Fix build on ARM, remove in next version
```

Reply:
yes, let me update the comment